core/src/main/scala/cats/Reducible.scala

```diff
@@ -44,13 +44,32 @@ import simulacrum.{noop, typeclass}
    * semigroup for `G[_]`.
    *
    * This method is a generalization of `reduce`.
+   *
+   * {{{
+   * scala> import cats.Reducible
+   * scala> import cats.data._
+   * scala> Reducible[NonEmptyVector].reduceK(NonEmptyVector.of(NonEmptyList.of(1, 2, 3), NonEmptyList.of(4, 5, 6), NonEmptyList.of(7, 8, 9)))
+   * res0: NonEmptyList[Int] = NonEmptyList(1, 2, 3, 4, 5, 6, 7, 8, 9)
+   * }}}
    */
   def reduceK[G[_], A](fga: F[G[A]])(implicit G: SemigroupK[G]): G[A] =
     reduce(fga)(G.algebra)
 
   /**
    * Apply `f` to each element of `fa` and combine them using the
    * given `Semigroup[B]`.
+   *
+   * {{{
+   * scala> import cats.Reducible
+   * scala> import cats.data.NonEmptyList
+   * scala> import cats.implicits._
```

Comment:
The `cats.implicits._` import is now deprecated.

```suggestion
   * scala> import cats.syntax.all._
```